README.md

```diff
@@ -65,7 +65,55 @@ Hence, we are trying to find and then follow good patterns in naming, code organ
 
 #### Releasing
 
-The standard release process is done manually.
+The standard release process is done manually. There are multiple
+repositories taking part in the process. The
+[renku](https://github.com/SwissDataScienceCenter/renku) project
+contains helm charts for deploying to kubernetes and the acceptance
+tests. The
+[terraform-renku](https://github.com/SwissDataScienceCenter/terraform-renku)
+project contains deployment descriptions for all environments.
+
+**renku-graph project:**
+- create a branch off the `origin/release` branch, name it
+  `prep-<version>`
+- either cherry-pick from development to here or do a `git rebase
+  origin/development`
+- make a PR for this branch to be merged into origin/release
+  ([example](https://github.com/SwissDataScienceCenter/renku-graph/pull/1095))
+- list some points in the description for what has been changed
+- merge the PR and create a release on github
+  - release description: list the points again but organized in
+    Features, Bugfixes etc
+    ([example](https://github.com/SwissDataScienceCenter/renku-graph/releases/tag/2.19.0))
+  - The release notes are public, so don't list refactorings.
+  - this will trigger actions that:
+    - Runs acceptance tests
+    - creates a PR on the
+      [renku](https://github.com/SwissDataScienceCenter/renku) project
+      to change the version in the `requirements.yml`
+      ([example](https://github.com/SwissDataScienceCenter/renku/pull/2704))
+
+**renku project:**
+- create a new entry in `CHANGELOG.rst` and amend it to the PR opened
+  by the bot
+- Merge the PR when satisfiled
```

Comment:
Maybe we could also mention somewhere that in a case a PR preparing a new release is already opened (e.g. https://github.com/SwissDataScienceCenter/renku/pull/2706) the steps are slightly different:
* update the graph version in the `requirements.yaml` on that opened PR directly
* add relevant info to the CHANGELOG on the PR as well.

Reply:
What would the github action do as a result of creating a release in `renku-graph`? Does it detect an existing PR if the version is the same? Otherwise we just close the obsolete PR I guess?

Reply:
I cannot say for sure what's going to happen. My point was about a slightly different situation, more or less what we are currently having on the renku repo. So UI released today so an auto-generated PR was created on renku for the UI. UI simply modified the PR and turned it into `Release 0.18.0`. When we'd release, an auto PR would be generated for graph but effectively we don't need the PR as we'd rather manually update the `Release 0.18.0` as I mentioned above.

Reply:
Understand, thank you. We then just close the other autogenerated PRs - I updated the notes in that regard as well.